pkg/validation/internal/operatorhub.go

```diff
@@ -71,15 +71,15 @@ import (
 // containing a list of categories will enable those categories to be used when comparing CSV categories for
 // OperatorHub validation. The json file should be in the following format:
 //
-//	```json
-//	{
-//		"categories":[
-//      "Cloud Pak",
-//      "Registry",
-//      "MyCoolThing",
-//  	 ]
-//	}
-// 	```
+//		```json
+//		{
+//			"categories":[
+//	     "Cloud Pak",
+//	     "Registry",
+//	     "MyCoolThing",
+//	 	 ]
+//		}
+//		```
```

Comment:
Could you explain these changes?

Reply:
I believe this also has to do with Go 1.19 formatting changes. All information on the changes for Go 1.19 doc comment changes should be able to be found here: https://tip.golang.org/doc/comment

Reply:
thank you @everettraven for your explanation.

Reply:
These changes are more due to 1.19 formatting changes.